src/app/page.module.css

```diff
@@ -1,9 +1,89 @@
-.main {
+.mainpageContainer {
+  background: #1b1228;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+
+.h1 {
+  width: 353px;
+  height: 92px;
+  top: 233px;
+  left: 19px;
+  gap: 0px;
+  opacity: 0px;
+  font-family: Cascadia Mono;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 46.48px;
+  text-align: center;
+  color: #ffffff;
+  margin-bottom: 100px;
+}
```

Comment:
you can just use the <h1> tag instead of defining a h1 class

Reply:
renaming the class to something like welcome text or heading will also be more readable than .h1